### PR TITLE
Update 999-big-list-of-options.md

### PR DESCRIPTION
**Pull request by aryzing**
Clarify role of build tools when handling module exports.

This PR contains:
- [ ] bugfix
- [ ] feature
- [ ] refactor
- [x] documentation
- [ ] other

Are tests included?
- [ ] yes (*bugfixes and features will not be merged without tests*)
- [x] no

Breaking Changes?
- [ ] yes (*breaking changes will not be merged unless absolutely necessary*)
- [x] no

List any relevant issue numbers:

### Description

Clarify that using `named` exports without build tools does not allow for equivalent ESM import signatures.

### CODE REVIEW

File: docs/999-big-list-of-options.md [hunk 1]
```diff
@@ -1130,7 +1130,7 @@ const {default: foo, bar} = require( "your-lib" );
 
 Note: There are some tools such as Babel, TypeScript, Webpack, and `@rollup/plugin-commonjs` that are capable of resolving a CommonJS `require(...)` call with an ES module. If you are generating CommonJS output that is meant to be interchangeable with ESM output for those tools, you should always use `named` export mode. The reason is that most of those tools will by default return the namespace of an ES module on `require` where the default export is the `.default` property.
 
-In other words for those tools, you cannot create a package interface where `const lib = require("your-lib")` yields the same as `import lib from "your-lib"`. With named export mode however, `const {lib} = require("your-lib")` will be equivalent to `import {lib} from "your-lib"`.
+Without those tools, you cannot create a package interface where `const lib = require("your-lib")` yields the same as `import lib from "your-lib"`. With named export mode however, `const {lib} = require("your-lib")` will be equivalent to `import {lib} from "your-lib"`.
```
Comment (lukastaegert) [inline docs/999-big-list-of-options.md:1133]: I agree that this is still confusing. However, the point is that even WITH those tools, you cannot create such an interface. If your package exports

```js
export default 'foo';
```

then

```js
import lib from "your-lib";
console.log(lib); // "foo"
```

while

```js
const lib = require("your-lib");
console.log(lib); // {default: "foo"}
```

This is what those tools will produce when there is only an ESM version of `your-lib`. Now if you also have a CommonJS version, then for equivalence, `require("your-lib")` should also return an object with a `default` property. This is what `"named"` export mode guarantees.

Maybe this can be made clearer with more examples?
